src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -61,8 +59,10 @@ public New3Command(string commandName, ITemplateEngineHost host, ITelemetryLogge
             CommandName = commandName;
             _paths = new Paths(EnvironmentSettings);
             _onFirstRun = onFirstRun;
-
+            _hostDataLoader = new HostSpecificDataLoader(EnvironmentSettings.SettingsLoader);
             _commandInput = commandInput;
+
+            EnvironmentSettings.Host.TryGetHostParamDefault("prefs:language", out _defaultLanguage);
```

Comment:
Explicitly set _defaultLanguage to an appropriate value (null) when TryGetHostParamDefault returns false